### PR TITLE
RBAC Testflows tests for ATTACH, CREATE, DROP, and DETACH

### PR DESCRIPTION
**Pull request by MyroTk**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
* Added RBAC tests for `ATTACH`, `CREATE`, `DROP`, and `DETACH`

### CODE REVIEW

**Comment by MyroTk** [comment]
@vitlibar Could you review and merge? The testflows check is green.

**Comment by vzakaznikov** [comment]
@vitlibar, Hi Vitaliy, this PR is ready for your review and merge. We have more tests coming and so we would like to have these merged as soon as possible if possible. Thanks!

File: tests/testflows/rbac/regression.py [hunk 1]
```diff
@@ -96,6 +96,12 @@
         [(Fail, issue_16403)],
     "privileges/alter move/:/:/:/:/user with revoked ALTER MOVE PARTITION privilege/":
         [(Fail, issue_16403)],
+    "privileges/create table/create with join query privilege granted directly or via role/:":
+        [(Fail, issue_14149)],
+    "privileges/create table/create with join union subquery privilege granted directly or via role/:":
+        [(Fail, issue_14149)],
+    "privileges/create table/create with nested tables privilege granted directly or via role/:":
+        [(Fail, issue_14149)],
```
Comment (vitlibar) [inline tests/testflows/rbac/regression.py:104]: The issue number seems to be wrong?

File: tests/testflows/rbac/requirements/requirements.md [hunk 2]
```diff
@@ -3186,14 +3226,14 @@ on tables created using the following engines
...
-##### AlterIndex
+##### Alter Index
 
 ###### RQ.SRS-006.RBAC.Privileges.AlterIndex
 version: 1.0
 
 [ClickHouse] SHALL support controlling access to the **alter index** privilege
 for a database or a specific table to one or more **users** or **roles**.
-Any `ALTER TABLE ... ORDER BY | ADD|DROP|MATERIALIZE|CLEAR INDEX` statements SHALL
+Any `ALTER TABLE ... ORDER BY | ADD|CREATE|MATERIALIZE|CLEAR INDEX` statements SHALL
```
Comment (vitlibar) [inline tests/testflows/rbac/requirements/requirements.md:3236]: We don't have `ALTER TABLE ... CREATE INDEX` command, but we have `ALTER TABLE ... DROP INDEX` command.

File: tests/testflows/rbac/requirements/requirements.md [hunk 3]
```diff
@@ -3162,7 +3202,7 @@ version: 1.0
 
 [ClickHouse] SHALL support granting or revoking **alter column** privilege
 on a specified cluster to one or more **users** or **roles**.
-Any `ALTER TABLE ... ADD|DROP|CLEAR|COMMENT|MODIFY COLUMN`
+Any `ALTER TABLE ... ADD|CREATE|CLEAR|COMMENT|MODIFY COLUMN`
```
Comment (vitlibar) [inline tests/testflows/rbac/requirements/requirements.md:3205]: We don't have `ALTER TABLE ... CREATE COLUMN` command, but we have `ALTER TABLE ... DROP COLUMN` command.

File: tests/testflows/rbac/requirements/requirements.md (hunk 2 above)
Comment (vitlibar) [reply tests/testflows/rbac/requirements/requirements.md:3236]: @MyroTk

File: tests/testflows/rbac/requirements/requirements.md (hunk 3 above)
Comment (vitlibar) [reply tests/testflows/rbac/requirements/requirements.md:3205]: @MyroTk

**Pull request merged**